lib/vssh/libssh.c

```diff
@@ -2570,7 +2579,43 @@ static ssize_t sftp_send(struct Curl_easy *data, int sockindex,
    */
   if(len > 32768)
     len = 32768;
+#if LIBSSH_VERSION_INT > SSH_VERSION_INT(0, 11, 0)
+  switch(conn->proto.sshc.sftp_send_state) {
+    case 0:
+        sftp_file_set_nonblocking(conn->proto.sshc.sftp_file);
```

Comment:
Indentation is 4 chars now for the code under the `case`s, while it should be 2.

Reply:
have correct the indentation.

Reply:
LGTM, thanks for the fixes!